src/pages/registration/description_tab/projects_field/ProjectsField.tsx

```diff
@@ -44,101 +50,217 @@ export const ProjectsField = () => {
+            <Box sx={{ display: 'flex', gap: '2rem' }}>
+              {project ? (
+                <>
+                  <Link
+                    data-testid={dataTestId.registrationWizard.description.projectLink(project?.id)}
+                    href={getProjectPath(project?.id ?? '')}
+                    target="_blank"
+                    rel="noopener noreferrer">
+                    {project?.title}
+                  </Link>
+                  <OpenInNewIcon fontSize="small" />
+                </>
+              ) : (
+                <ListSkeleton arrayLength={1} minWidth={20} height={20} />
+              )}
+            </Box>
+          </div>
+          <div>
+            <Typography fontWeight="bold">{t('project.coordinating_institution')}:</Typography>
+            {project?.coordinatingInstitution ? (
+              <Typography>{getLanguageString(project?.coordinatingInstitution.labels)}</Typography>
+            ) : (
+              <ListSkeleton arrayLength={1} minWidth={20} height={20} />
+            )}
+          </div>
+        </Box>
+      </Box>
+
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+        <Typography fontWeight="bold">{t('common.funding').toUpperCase()}</Typography>
+        {project?.funding?.length ? (
+          project.funding.map((funding) => (
+            <Box
+              key={funding.identifier}
+              sx={{ display: 'grid', gridTemplateRows: 'auto auto', gap: '1rem', height: '100%' }}>
+              <div>
+                <Typography fontWeight="bold">{t('registration.description.funding.funder')}:</Typography>
+                <Typography>{getLanguageString(funding.labels)}</Typography>
+              </div>
+              <div>
+                <Typography fontWeight="bold">{t('project.grant_id')}:</Typography>
+                {funding.identifier ? (
+                  fundingSourceIsNfr(funding.source) ? (
+                    <Box sx={{ display: 'flex', gap: '2rem', height: 'fit-content' }}>
+                      <Link
+                        data-testid={dataTestId.registrationWizard.description.nfrProjectLink(funding.identifier)}
+                        href={getNfrProjectUrl(funding.identifier)}
+                        target="_blank"
+                        rel="noopener noreferrer">
+                        {funding.identifier}
+                      </Link>
+                      <OpenInNewIcon fontSize="small" />
+                    </Box>
+                  ) : (
+                    <Typography>{funding.identifier}</Typography>
+                  )
+                ) : (
+                  <ListSkeleton arrayLength={1} minWidth={20} height={20} />
+                )}
+              </div>
+            </Box>
+          ))
+        ) : (
+          <Typography fontStyle="italic">{t('project.project_has_no_funding')}</Typography>
+        )}
+      </Box>
+      <IconButton
+        size="small"
```

Comment:
Ble en ganske stor knapp
![bilde](https://github.com/user-attachments/assets/2743b4ea-c742-49eb-ba7d-960663643a97)

```suggestion
        sx={{ alignSelf: 'center' }}
```